packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -357,9 +357,11 @@ export const useChatMessageCommonStyles = makeStyles({
   },
   bodyWithPlaceholderImage: {
     '& img[src=""]': {
-      display: 'block',
+      fontSize: '0',
```

Comment:
Could you add a comment why we need to set this?

Reply:
added thanks